package/gluon-scheduled-domain-switch/check_site.lua

```diff
@@ -0,0 +1,6 @@
+if need_table(in_domain({'domain_switch'}), check_domain_switch, false) then
+	need_domain_name(in_domain({'domain_switch', 'target_domain'}), true)
+	need_number(in_domain({'domain_switch', 'switch_after_offline_mins'}), true)
+	need_number(in_domain({'domain_switch', 'switch_time'}), true)
+	need_string_array_match(in_domain({'domain_switch', 'connection_check_targets'}), '^[%x:]+$', true)
```

Comment:
It's common to leave out the *required* argument when it is the default (true).

Reply:
removed the explicit `true`